CMakeLists.txt

```diff
@@ -291,6 +291,11 @@ IF (NOT KOKKOS_HAS_TRILINOS AND NOT Kokkos_INSTALL_TESTING)
 ENDIF()
 INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_install.cmake)
 
+IF (Kokkos_ENABLE_HIP)
+  FIND_PACKAGE(rocthrust)
+  TARGET_LINK_LIBRARIES(kokkos INTERFACE roc::rocthrust)
+ENDIF()
```

Comment:
What would be a good place for this find_package? Do we want an additional option (maybe Kokkos_ENABLE_ROCTHRUST) enabling the usage of ROCm Thrust?